core/templates/pages/topics-and-skills-dashboard-page/navbar/topics-and-skills-dashboard-navbar.directive.ts

```diff
@@ -77,21 +77,21 @@ angular.module('oppia').directive('topicsAndSkillsDashboardNavbar', [
                   };
                   var newExplanationObject = null;
 
-                  $scope.$watch('newSkillDescription', function() {
-                    $scope.rubrics[1].setExplanation(
-                      '<p>' + $scope.newSkillDescription + '</p>');
-                  });
+                  // $scope.$watch('newSkillDescription', function() {
```

Comment:
@seanlip This part here was to automatically update the 'Medium' explanation with skill description, but now that the explanations are a list, is this still needed (like maybe have one explanation for medium initially and autofill there)?